docs/advanced_guides/using_agenta_from_cli.mdx

```diff
@@ -45,8 +45,35 @@ The CLI will also display the URL of the endpoint, which can be used to test the
 ## Adding a new configuration
 
 In addition to the default configuration specified in the code, you can add more configurations to the application from the CLI.
+Configurations are specified in toml files and always named `<codebase_name>.<configuration_name>`.
+> When you serve your first app variant, the configuration of the created app variant is pulled from the Backend and saved in a toml file named `<codebase_name>.<configuration_name>.toml`
+>
+> <img width="239" alt="image" src="https://github.com/Agenta-AI/agenta/assets/56418363/da80dd1b-c308-4ad0-baea-396ae727ba77" /><img width="700" alt="image" src="https://github.com/Agenta-AI/agenta/assets/56418363/d70cfa07-dc9a-4486-8090-54ba5a32bd3e" />
```

Comment:
@devgenix I don't understand the content of the config.toml??
Why do we have config_id, current_version, created_at, version history, and lots and lots of useless stuff. It is supposed to be a human readable format to specify the configuration.

Reply:
@mmabrouk Oh, initially I had it in mind to save only the parameters gotten from the configuration but when I asked you mentioned that I pull and save everything; https://agenta-hq.slack.com/archives/D05MTU0FWH2/p1703016946840149?thread_ts=1703013964.710329&cid=D05MTU0FWH2

But what do you have in mind that the configuration are formatted?